index.html

```diff
@@ -7,6 +7,116 @@
   <body>
 
     <p>Репозиторий создан для обучения на профессиональном онлайн‑курсе «<a href="https://htmlacademy.ru/intensive/htmlcss">HTML и CSS, уровень 1</a>».</p>
+    <header>
+      <nav>
+        <a href="#">
```

Comment:
```suggestion
        <a>
```
Страница не должна на себя ссылаться.